doc/releases/changelog-0.35.0.md

```diff
@@ -6,20 +6,150 @@
+  [QuantumCircuit](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit) into
+  a PennyLane
+  [quantum function](https://docs.pennylane.ai/en/stable/introduction/circuits.html#quantum-functions).
+  Although `qml.from_qiskit` already exists in PennyLane, we have made a number of improvements to
+  make importing from Qiskit easier:
   [(#5218)](https://github.com/PennyLaneAI/pennylane/pull/5218)
+  [(#5168)](https://github.com/PennyLaneAI/pennylane/pull/5168)
+
+  * You can now append PennyLane measurements onto the quantum function returned by
+    `qml.from_qiskit`. Consider this simple Qiskit circuit:
+
+    ```python
+    import pennylane as qml
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.rx(0.785, 0)
+    qc.ry(1.57, 1)
+    ```
+    
+    We can convert into a PennyLane QNode in just a few lines.
+
+    ```pycon
+    >>> dev = qml.device("default.qubit")
+    >>> measurements = [qml.expval(qml.Z(0) @ qml.Z(1))]
+    >>> qfunc = qml.from_qiskit(qc, measurements=measurements)
+    >>> qnode = qml.QNode(qfunc, dev)
+    >>> qnode()
+    [tensor(0.00056331, requires_grad=True)]
+    ```
+
+  * Quantum circuits that already contain Qiskit-side measurements can be faithfully converted with
+    `qml.from_qiskit`:
+
+    ```python
+    qc = QuantumCircuit(3, 2)  # Teleportation
+  
+    qc.rx(0.9, 0)  # Prepare input state on qubit 0
+  
+    qc.h(1)  # Prepare Bell state on qubits 1 and 2
+    qc.cx(1, 2)
+  
+    qc.cx(0, 1)  # Perform teleportation
+    qc.h(0)
+    qc.measure(0, 0)
+    qc.measure(1, 1)
+  
+    with qc.if_test((1, 1)):  # Perform first conditional
+        qc.x(2)
+    ```
+    
+    This circuit can be converted into PennyLane with the mid-circuit measurements still accessible.
+
+    ```python
+    @qml.qnode(dev)
+    def teleport():
+        m0, m1 = qml.from_qiskit(qc)()
+        qml.cond(m0, qml.CZ)(2)
+        return qml.density_matrix(2)
+    ```
+    
+  * It is now more intuitive to handle parametrized Qiskit circuits. Consider the following circuit:
 
-* A Qiskit `SparsePauliOp` can be converted into a PennyLane `Operator` using `qml.from_qiskit_op`.
+    ```python
+    from qiskit.circuit import Parameter
+
+    angle0 = Parameter("x")
+    angle1 = Parameter("y")
+
+    qc = QuantumCircuit(2, 2)
+    qc.rx(angle0, 0)
+    qc.ry(angle1, 1)
+    qc.cx(1, 0)
+    ```
+    
+    We can convert this circuit into a QNode with two arguments, corresponding to `x` and `y`:
+
+    ```pycon
+    measurements=qml.expval(qml.PauliZ(0))
+    qfunc = qml.from_qiskit(qc, measurements)
+    qnode = qml.QNode(qfunc, dev)
+    ```
+    
+    The QNode can be evaluated and differentiated:
+
+    ```pycon
+    >>> x, y = qml.numpy.array([0.4, 0.5], requires_grad=True)
+    >>> qnode(x, y)
+    tensor(0.80830707, requires_grad=True)
+    >>> qml.grad(qnode)(x, y)
+    (tensor(-0.34174675, requires_grad=True),
+     tensor(-0.44158016, requires_grad=True))
+    ```
```

Comment:
Still not convinced that we should avoid `qml.QNode` :thinking: It's in [our docs](https://docs.pennylane.ai/en/stable/introduction/circuits.html#creating-a-quantum-node) and I feel like I've seen it used by users before.

Reply:
I'll think on it 🤔 ... the perspective of a qiskit user has never entered my mind before lol. So, indeed, it might be better to do `qml.QNode(qfunc, dev)` from their POV